### ISSUE TITLE
randon result for multiple trials with the same input image

### ISSUE THREAD

**Issue by qyjyork** [facebookresearch/dinov2#551]
i was using dinov2 model to get the feature result for an image. yet when i run the code for several time (restart for every time), the feature result is different. even if i do not restart the code and analyze the same image for two successive times, results are still but slightly different. i wonder if there's random seed config somewhere yet i cannot find it.
the reason i need consistency is that i want to store the feature results of different images into local files. and when i want to compare them, i only need to load the stored features rather than go-through dinov2 again.

**Comment by baldassarreFe** [comment]
To help you debug:
- When you load the model, do you call `model.eval()` to disable all random operations like dropout and stochastic drop path?
- Can you print the weights of the model? Are they the same across restarts or do they change? Maybe there is an issue and they are not loaded.

Please share the code for a minimal reproduction example so we can have a look.
